umpf

```diff
@@ -1414,6 +1414,13 @@ format_patch_end() {
 	if ${NIX}; then
 		echo "]" >&${series_out}
 	fi
+	if ${BB}; then
+		local base_name="$(<"${STATE}/base-name")"
+		echo "UMPF_BASE = \"${base_name//v/}\"" >&${series_out}
+		echo "UMPF_VERSION = \"$(<"${STATE}/version")\"" >&${series_out}
+		echo "PV = \"\${UMPF_BASE}-\${UMPF_VERSION}\"" >&${series_out}
+		echo "LINUX_VERSION = \"\${UMPF_BASE}\"" >&${series_out}
```

Comment:
```suggestion
		if [ "$(head -n 1 README 2>/dev/null)" = "Linux kernel" ]; then
			echo "LINUX_VERSION = \"\${UMPF_BASE}\"" >&${series_out}
		fi
```
Works with Linux Kernel Versions >= 4.10 and I don't think there will be false positives with other packages

Reply:
Looks good and works. Did not have in mind that we have to be in the project's source dir anyway :smirk:

Reply:
We need that anyways to patch the Makefile etc...